client/options.go

```diff
@@ -414,12 +414,14 @@ func newDefaultCallOptions() *CallOptions {
 	}
 }
 
+// WithCallRequestTimeout the maximum waiting time for one specific call, only works for 'tri' and 'dubbo' protocol
 func WithCallRequestTimeout(timeout string) CallOption {
```

Comment:
> 1. Should we modify **WithCallRequestTimeout** to **WithCall_RequestTimeout** because there are **WithServer_XXX** options in server module? Or just modify **WithServer_XXX** to **WithServerXXX** for unification?

I think 'just modify **WithServer_XXX** to **WithServerXXX** for unification' is better, to keep consistent with  the `dubbo` package.